virttest/utils_test/libvirt.py

```diff
@@ -1477,9 +1476,8 @@ def create_hostdev_xml(pci_id, boot_order=None, xmlfile=True,
                     target=id_parts[2], unit=id_parts[3])))
 
     logging.debug("Hostdev XML:\n%s", str(hostdev_xml))
-    if not xmlfile:
-        return hostdev_xml
-    return hostdev_xml.xml
```

Comment:
May I ask what the issue is? Can you please describe the problem or paste the error message? Do we need to wait a while to make sure the file is created?

Reply:
Sometimes cases will get below error:
```
rhel.virsh.detach_device_alias.current.hostdev
CmdError: Command '/bin/virsh attach-device avocado-vt-vm1 /tmp/xml_utils_temp_9exbbws7.xml --current' failed.stdout: b'\n'stderr: b"error: Failed to open file '/tmp/xml_utils_temp_9exbbws7.xml': No such file or directory\n"additional_info: None
```

Directly returning object.xml might be unstable.